src/rotest/api/common/responses.py

```diff
@@ -0,0 +1,52 @@
+"""Responses models of the view requests."""
+from swaggapi.api.builder.common.response import AbstractResponse
+from swaggapi.api.builder.common.fields import (StringField,
+                                                ModelField,
+                                                ArrayField, BoolField)
+
+from rotest.api.common import GenericModel
+
+
+class SuccessResponse(AbstractResponse):
+    """Returns when request ended successfully - with no content."""
+    PROPERTIES = []
+
+
+class TokenResponseModel(AbstractResponse):
+    """Returns a token of the created sessions."""
+    TITLE = "Token Response"
+    PROPERTIES = [
+        StringField(name="token", description="Token of the current session",
+                    required=True)
+    ]
+
+
+class CleanupUserResponseModel(AbstractResponse):
+    """Return details about the executed request."""
+    PROPERTIES = [
+        StringField(name="details", required=True)
+    ]
+
+
+class BadRequestResponseModel(AbstractResponse):
```

Comment:
There's no special status code like 400?

Reply:
I think BadRequest is not right here.
we send it when the resources are locked, for example.
maybe FailureResponseModel would be better?

Reply:
@osherdp  the status code is irrelavant to the content sent to the user, therefore we  need to specify the structure of the response.
@UnDarkle I can relate to that.